algorithms/hhl/hhl.ipynb

```diff
@@ -0,0 +1,1367 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        global_phase = np.angle(qsol)
</pre><p>You should take the global phase from one component, say <code>global_phase = np.angle(qsol[0])</code> , and normalize by it. Note that normalizing by a global phase still might result in a minus sign difference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Modified `global_phase` and comment in the code and added note about sign ambiguity.